src/main/java/frc/robot/commands/Teleop/SensorIntake.java

```diff
@@ -25,12 +28,20 @@ public void initialize(){
     }
 
     public void execute() {
-        measurement = laserCanSensor.getMeasurement().distance_mm;
-        intakeSubsystem.setIntakeSpeed(Constants.Speed.INTAKE);
+        measurement1 = laserCanSensor1.getMeasurement().distance_mm;
```

Comment:
Can we just keep these as values local to the execute() and isFinished() methods instead of storing them in class variables?